### ISSUE TITLE
Xception 모델 성능 개선을 위한 단계별 기법 적용

### ISSUE THREAD

**Issue by Ryuchanghoon** [LoadKill/Overloaded-Detection#15]
기존 Xception 모델 [#14](https://github.com/orgs/LoadKill/projects/1?pane=issue&itemId=112138764&issue=LoadKill%7COverloaded-Detection%7C14)
300 epoch 학습 결과, Epoch 24에서 최고 정확도 보이는 상황.
이후 train loss는 지속 하락, val loss는 불안정하게 증가 ==> **과적합 현상** 발생.

</br>

## 실험 구성

|단계|실험|적용 기법|목적|
|:---:|:---:|:---:|:---:|
|1단계|Baseline 개선|`BCEWithLogitsLoss` </br> `밤 환경 고려 학습 데이터 증강` </br> `파라미터 값 조정`|안정화 확인 용도|
|2단계|과적합 완화 기법 적용|`Dropout` </br> `Label Smoothing` </br> `Weight Decay` </br> (위 기법 포함)|과적합 완화 및 성능 향상|

**Comment by Ryuchanghoon** [comment]
## xception 두번째 단계

- .pth 파일 시점의 정확도: 88.22% (최고 분류 정확도)
- batch_size = 64
- Epochs = 70
- 해당 정확도 나온 epoch: 62

</br>

![Image](https://github.com/user-attachments/assets/d64b1f6f-c607-4b32-8e2f-ea1a8d464278)

**Comment by Ryuchanghoon** [comment]
## 최종 xception 학습 적용

- .pth 파일 시점의 정확도: 90.06% (최고 분류 정확도)
- batch_size = 64
- Epochs = 200
- 해당 정확도 나온 epoch: 20

</br>

![Image](https://github.com/user-attachments/assets/88e2d104-e6a0-428e-901b-c290e1fa3556)
